src/Day4/Program2/tweet.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Created by TTND on 12/27/2016.
+ * Create components Cart, Item and CartTotal. Cart component displays list of Items and CartTotal. (set items in the state of Cart)  sample items = [ {name: 'item1', qty:2, price: 30}, {name: 'item2', qty:5, price: 10}, {name: 'item3', qty:7, price: 120}, ]
+ */
+import React from 'react';
+export default class Tweet extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: '',
+            limit:'150',
+            charleft:'150',
+            event: 'none',
+            background: 'grey'
+
+        };
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+    handleChange(event) {
+
+        if(event.target.value.length <1 || event.target.value.length >= 150) {
+            this.setState({
+                event: 'none',
+                background:'grey',
+                value: event.target.value,
+                charleft: this.state.limit - event.target.value.length
+            });
+        }
+        else {
+            this.setState({
+                event: 'inherit',
```

Comment:
To set default pointer-events use ```auto ``` as ```inherit``` will use pointer-events of parent element.